README.md

```diff
@@ -89,6 +90,7 @@ Here's a rough (and potentially incorrect) list of default ports for the various
 | `SevenDays` | UDP 26900 to UDP 26902 & TCP 26900 ([Guide](https://developer.valvesoftware.com/wiki/7_Days_to_Die_Dedicated_Server))                                                                          |
 | `srcds`     | Depends on the game. Usually UDP 27015. ([List of games under srcds](https://github.com/CubeCoders/AMP/wiki/Supported-Applications-Compatibility#applications-running-under-the-srcds-module)) |
 | `StarBound` | TCP 21025 ([Guide](https://starbounder.org/Guide:Setting_Up_Multiplayer))                                                                                                                      |
+| `Valheim`   | UDP 5678 → 5680                                                                                                                                                                                |
```

Comment:
Worth noting that these seem to be the default ports AMP uses, but aren't the default Valheim ports.